guides/source/installing_ruby.md

```diff
@@ -0,0 +1,105 @@
+**DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON https://guides.rubyonrails.org.**
+
+Installing Ruby Guide
+=====================
+
+This guide will walk you through installing the Ruby programming language on your operating system.
+
+While your OS might come with Ruby pre-installed, it's often outdated and can't be upgraded. Using a version manager like [Mise](https://mise.jdx.dev/getting-started.html) allows you to install the latest Ruby version, use a different Ruby version for each app, and easily upgrade to new versions when they're released.
+
+--------------------------------------------------------------------------------
+
+## Choose your operating system
+
+Follow the section for the operating system you use:
+
+* [macOS](#installing-ruby-on-macos)
+* [Ubuntu](#installing-ruby-on-ubuntu)
+* [Windows](#installing-ruby-on-windows)
+
+NOTE: Any commands prefaced with a dollar sign `$` should be run in the command line.
+
+### Installing Ruby on macOS
+
+For macOS, you'll need Xcode Command Line Tools and Homebrew to install dependencies needed to compile Ruby.
```

Comment:
macOS Catalina (10.15) switched the default shell to zsh, so we should note that here.